src/amo/reducers/redirectTo.js

```diff
@@ -0,0 +1,60 @@
+/* @flow */
+
+const SEND_SERVER_REDIRECT: 'SEND_SERVER_REDIRECT' = 'SEND_SERVER_REDIRECT';
+
+type State = {|
+  url: null | string,
+  status: null | 301 | 302 | 303 | 305 | 307 | 308,
+|};
+
+export const initialState = {
+  url: null,
+  status: null,
+};
+
+type SendServerRedirectParams = {|
+  ...State,
+|};
+
+type SendServerRedirectAction ={|
+  type: typeof SEND_SERVER_REDIRECT,
+  payload: SendServerRedirectParams,
+|};
+
+export const sendServerRedirect = (
+  { status, url }: SendServerRedirectParams
+): SendServerRedirectAction => {
+  if (!status) {
+    throw new Error('status is required');
+  }
+  if (!url) {
+    throw new Error('url is required');
+  }
```

Comment:
I like how you put "server" in the name. It would also be nice if this had some extra logging when called client side -- just another thing to dissuade someone from using it or to catch bugs. 

```js
if (!config.get('server')) {
  log.warn('sendServerRedirect() currently does nothing when run from client code');
}
```

No one should ever see this message (hopefully!).